README.md

```diff
@@ -21,14 +21,14 @@ the following instructions to find or create these values if necessary.
     * `AZ_CLIENT_ID`*
     * `AZ_CLIENT_SECRET`*
```

Comment:
is this mandatory ?

Reply:
When authenticating using a service principal, yes